### PR TITLE
finally makes a bit of good on the species restricted syndicate items

### PR DESCRIPTION
**Pull request by dinnerwoerror**
## About The Pull Request

i'm adding a bunch of species restricted traitor items and also changing some of the shitty ones

if you wanna give me ideas for like, spdiers or cats or something i donno i already made this one

**the folding shotgun**

the legend goes that the creator of this ancient marvel of engineering started out by making an appointment with the NYPD commissioner Theodore Roosevelt, and arrived with one of these folding shotguns under his coat, and after exchanging a few greetings, the inventor thoroughly startled Teddy Roosevelt by whipping out the gun from its holster, unfolding it with a loud snap, and blasting six shots into the ceiling of Roosevelt's office over the course of about eight or ten seconds, and Roosevelt, always one to be enthusiastic about guns, promptly ordered 100 of them to be used in the New York State Penal System

listen, it's an American thing, ok? you wouldn't understand it at first, and i didn't either, but i eventually came around to it and you will too, eventually

holster

**moth conversion chemicals**

there is no moth hivemind

**boogie bomb**

changeling: Nobody can stop me
A stray boogie bomb: I'm about to end this man's whoole career

sadly you cannot glitch infinite items into your backpack with this

**Punisher's envirosuit**

i think this one deserves no explanation

it was made as an analogue to the infiltrator kit which can't be worn by plasmamen, so if you want to howl about bAlAnCe, compare it to that

## Why It's Good For The Game

i figured someone might as well get around to it

all of these are kind of shit but its my shit which is why i shill it

## Changelog
:cl:
add: fleshed out the species restricted big bad items finally
/:cl:

### CODE REVIEW

File: waspstation/code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -28,3 +28,21 @@
...
+	desc = "A custom made plasmaman containment suit designed from a head of security's envirosuit, with devilstrand silk, very tough, good at insulating, and protects exceptionally well."
+	icon = 'waspstation/icons/obj/clothing/under/syndicate.dmi'
+	mob_overlay_icon = 'waspstation/icons/mob/clothing/under/syndicate.dmi'
+	icon_state = "punisher_envirosuit"
+	item_state = "punisher_envirosuit"
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 100, "rad" = 0, "fire" = 100, "acid" = 100)
+
+/obj/item/clothing/head/helmet/space/plasmaman/punisher
+	name = "Executioners' plasma envirosuit helmet"
+	desc = "Those who do wrong to others...the traitors, the changelings, spies, tiders...you will come to know me well. Oganesson CXVIII is dead. Call me...the Executioner."
```
Comment (Superyodeler) [inline waspstation/code/modules/clothing/under/syndicate.dm:43]: No character references.

Comment (Maldaris) [reply waspstation/code/modules/clothing/under/syndicate.dm:43]: doubly so for incredibly edgy character references

**Comment by SynnGraffkin** [comment]
Cringe

**Comment by tiramisuapimancer** [comment]
dance grenade's uplink description conflicts with ethereal lore, they wouldn't have testosterone

**Comment by SynnGraffkin** [comment]
Ethereals have lore?

**Comment by triplezeta** [comment]
arent ethereals genderless

why would they have testosterone

**Comment by tiramisuapimancer** [comment]
they are also aliens

**Comment by Superyodeler** [comment]
Testosterone wouldn't really make sense due to the whole energy alien thing, but both men and women have testosterone, men just have a lot more of it.

**Pull request closed without merging**